pkg/apis/playlist/v0alpha1/legacy_storage.go

```diff
@@ -87,7 +83,10 @@ func (s *legacyStorage) List(ctx context.Context, options *internalversion.ListO
 				APIVersion: APIVersion,
 			},
 			ObjectMeta: metav1.ObjectMeta{
-				Name: v.UID,
+				Name:              v.UID,
+				Namespace:         s.mapper(v.OrgId),
```

Comment:
🤔 this is only really necessary if the list can return values from a namespace other than the raw request.  But i think worth the effort so we do not blindly replace them from the request.